SwiftCharts/Axis/ChartAxisLayerDefault.swift

```diff
@@ -35,6 +74,7 @@ public class ChartAxisSettings {
     }
 }
 
+/// A default class that implements the ChartAxisLayer protocol
```

Comment:
Maybe something a bit more detailed like "Default implementation of ChartAxisLayer, which renders axis line and labels using Core Graphics"?